### ISSUE TITLE
🐛 Unexpected jitter on `MenuFlyoutSubItem`  appears

### ISSUE THREAD

**Issue by TENX-S** [bdlukaa/fluent_ui#1014]
When the `MenuFlyoutSubItem` appears, it quickly moves a short distance to the lower right corner.

https://github.com/bdlukaa/fluent_ui/assets/40336192/8d3a0592-5ce8-4058-b8ab-de450ca35bec

**Comment by TENX-S** [comment]
This behavior can be confirmed in example app too.
![image](https://github.com/bdlukaa/fluent_ui/assets/40336192/ad80397d-1856-4668-98f4-2a4f0d89cb1f)

**Comment by TENX-S** [comment]
Yeah. I think that is the root cause. I build and run the example without the post frame callback, and the jitter is gone. But the `FlyoutListTile` size doesn't get right:
<img width="222" alt="image" src="https://github.com/bdlukaa/fluent_ui/assets/40336192/14714e77-fb4a-4652-89f4-1f4a8b1bdca6">

**Comment by bdlukaa** [comment]
Yeah. To fix this, we'd need to make use of Flutter box system - which is kind of complex. We already do it for other widgets, like `ComboBox`.
